pages/sites.tsx

```diff
@@ -37,11 +38,20 @@ const Sites = () => {
     <div className="h-full w-full flex flex-col gap-3 items-center px-5">
       <div className="flex flex-row w-full h-12 items-end mb-4 max-w-lg ">
         <h1 className="flex-1 font-bold text-3xl text-ocf-gray">My Sites</h1>
-        <button onClick={() => setEditMode(!editMode)}>
+        <button
+          onClick={() => {
+            setEditMode(!editMode);
+            setIsDisabled(true);
+            setTimeout(() => {
+              setIsDisabled(false);
+            }, 900);
```

Comment:
lets set this as a constant up above this component, and export it from the file! that way, we can access it wherever else we'll need the transition duration! 🔥